code/game/objects/items/devices/scanners.dm

```diff
@@ -227,15 +227,19 @@ REAGENT SCANNER
 				user.show_message(text("\red Internal bleeding detected. Advanced scanner required for location."), 1)
 				break
 		if(H.vessel)
-			var/blood_volume = round(M:vessel.get_reagent_amount("blood"))

```

Comment:
Exact same thing was done in healthanalyzer code too. I really don't understand what went through the minds of oldcoders.